pyrealm/hygro.py

```diff
@@ -37,15 +37,15 @@ def calc_vp_sat(ta: NDArray, const: HygroConst = HygroConst()) -> NDArray:
         >>> # Saturated vapour pressure at 21°C
         >>> import numpy as np
         >>> temp = np.array([21])
-        >>> round(calc_vp_sat(temp), 6)
-        2.480904
+        >>> calc_vp_sat(temp).round(6)
+        array([2.480904])
         >>> from pyrealm.constants import HygroConst
         >>> allen = HygroConst(magnus_option='Allen1998')
-        >>> round(calc_vp_sat(temp, const=allen), 6)
-        2.487005
+        >>> calc_vp_sat(temp, const=allen).round(6)
```

Comment:
Example of where the help text misses the import. Although I suspect it's likely that the user could recover (user needs to understand the context from which they are accessing the method i.e. maybe they should use pyrealm.hygro.calc_vp_sat(args).